### PR TITLE
Updating logon event

### PR DESCRIPTION
**Pull request by joinimran**
In this page, logon events were listed for Windows Xp, 7. I have updated them to Windows 10.

Problem: https://github.com/MicrosoftDocs/windows-itpro-docs/issues/7178

### CODE REVIEW

**Comment by joinimran** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

File: windows/security/threat-protection/auditing/basic-audit-logon-events.md [hunk 1]
```diff
@@ -38,33 +38,12 @@ You can configure this security setting by opening the appropriate policy under
...
+| 4625          | Logon failure. A logon attempt was made with an unknown user name or a known user name with a bad password.                                                                                                     |
+| 4634          | The logoff process was completed for a user.                                                                                                                                                                    |                                                                                                                               
+| 4647          | A user initiated the logoff process.                                                                                                                                                                            |
+| 4648         | A user successfully logged on to a computer using explicit credentials while already logged on as a different user.                                                                                             |
```
Comment (illfated) [inline windows/security/threat-protection/auditing/basic-audit-logon-events.md:45]: Cosmetic/codestyle: use the same indent as the other lines:
```suggestion
| 4648          | A user successfully logged on to a computer using explicit credentials while already logged on as a different user.                                                                                             |
```

**Review by illfated — APPROVED** [review]
Thank you. Looks good to me as well.

**Comment by illfated** [comment]
#sign-off

**Pull request merged**